### PR TITLE
Completed layout classes

### PR DESCRIPTION
**Pull request by chrisahn1**
css layout classes

### CODE REVIEW

File: css-layout-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+            </div>
+
+          </article>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-full">
+          <footer></footer>
+        </div>
+      </div>
```
Comment (Zhiwutian) [inline css-layout-classes/index.html:100]: There seems to be so extra HTML here, I would remove any that is not being used

File: css-layout-classes/index.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        <div class="column-full">
+          <header></header>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-one-third">
+          <aside></aside>
+        </div>
+        <div class="column-two-thirds">
+          <article>
```
Comment (Zhiwutian) [inline css-layout-classes/index.html:18]: same as below, this appears to be unrelated HTML, make sure to remove any which is not being used.

File: css-layout-classes/styles.css [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-one-third {
```
Comment (Zhiwutian) [inline css-layout-classes/styles.css:13]: make sure your column classes have appropriate widths set on them. and that you have them for all of the column classes.

File: css-layout-classes/styles.css [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  width: 60%;
+}
+
+.summary-dk {
```
Comment (Zhiwutian) [inline css-layout-classes/styles.css:17]: we don't want to use auto width anywhere is this exercise, instead, make sure to set the appropriate widths as needed to the column classes.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Good start! A few updates to go! Reach out if you have questions!

**Comment by chrisahn1** [comment]
css layout classes 2

**Review by Zhiwutian — APPROVED** [review]
👍🏼
